RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -57,18 +57,21 @@
 RecoLocalCaloRECO.outputCommands.extend(ecalLocalRecoRECO.outputCommands)
 RecoLocalCaloAOD.outputCommands.extend(ecalLocalRecoAOD.outputCommands)
 
-def _modifyRecoLocalCaloEventContentForHGCalRECO( obj ):
-    obj.outputCommands.append('keep *_HGCalRecHit_*_*')
+# mods for HGCAL
 
-def _modifyRecoLocalCaloEventContentForHGCalFEVT( obj ):
-    obj.outputCommands.append('keep *_HGCalRecHit_*_*')
-    obj.outputCommands.append('keep *_HGCalUncalibRecHit_*_*')
+_phase2_RecoLocalCaloFEVT_outputCommands = RecoLocalCaloFEVT.outputCommands
+_phase2_RecoLocalCaloFEVT_outputCommands.append('keep *_HGCalRecHit_*_*')
+_phase2_RecoLocalCaloFEVT_outputCommands.append('keep *_HGCalUncalibRecHit_*_*')
```

Comment:
@kpedro88 My main concern is that does the assignment `_phase2_RecoLocalCaloFEVT_outputCommands = RecoLocalCaloFEVT.outputCommands` create a copy or not (I fear not). A simplified example
```
import FWCore.ParameterSet.Config as cms
foo = cms.vstring("foo")
bar = foo # corresponds "_phase2_RecoLocalCaloFEVT_outputCommands = RecoLocalCaloFEVT.outputCommands"
bar.append("bar")
print foo
# prints "foo" and "bar"
```

Temporary object I think it's more a matter of style, I mostly wanted to give an example how to avoid it.